paket.dependencies

```diff
@@ -65,7 +65,8 @@ nuget BenchmarkDotNet.Diagnostics.Windows 0.13.1
 group Build
     source https://api.nuget.org/v3/index.json
     storage: none
-    nuget FSharp.Core
+
+    nuget FSharp.Core ~> 7
```

Comment:
this is the material change - v7 and later don't have erroneous xmldoc as content in the package
can alternately override with a `content: none`

Reply:
paket changes have been removed as they were pretty much there in v5